examples/cms/src/SelectDataSource.tsx

```diff
@@ -0,0 +1,73 @@
+import { framer } from "framer-plugin"
+import { useState } from "react"
+import { Spinner } from "./components/Spinner"
+import { DataSource, getDataSource } from "./data"
+
+function Logo() {
+    return (
+        <div className="logo">
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 150 150">
+                <path
+                    fill="#999"
+                    d="M75 33c18.778 0 34 7.611 34 17S93.778 67 75 67s-34-7.611-34-17 15.222-17 34-17Zm34 40.333C109 82.538 93.778 90 75 90s-34-7.462-34-16.667V60c0 9.389 15.222 17 34 17 18.776 0 33.997-7.61 34-16.997v13.33Z"
+                />
+            </svg>
+        </div>
+    )
+}
```

Comment:
I would make icon components just the SVG, no other HTML tags. Logo is a generic name, maybe the name of the component could be more descriptive so users will recognize it.